### DISCUSSION TITLE
Failed to open/installing Halide on windows 10

### DISCUSSION THREAD

**Discussion by Rohan-Zanjal** [halide/Halide#5835] [Q&A] [answered]
![Screenshot (13)](https://user-images.githubusercontent.com/71709895/112009856-ca230480-8b4c-11eb-89cb-3a9af1fa08fb.png)

I have Vs code, But iam facing this Iissue , 
can any one solve issues

**Comment by alexreinking** [comment, marked as answer]
The error message very clearly states that you do not have a suitable (I assume this means "new enough") Visual Studio installed. It is also clear from the output that it failed when trying to build _zlib_, not _Halide_, so this is not an issue we can help you solve. If you actually do have Visual Studio installed, then the problem is with vcpkg and you should open an issue with them, here: https://github.com/microsoft/vcpkg

**Reply by Rohan-Zanjal** [reply]
So can I Reinstall Visual Studio on my system

**Reply by alexreinking** [reply]
I think you need to install it _in the first place_. I see that you have Visual Studio **Code** installed. That is not the same thing as Visual Studio. You can get Visual Studio 2019 Community Edition from here: https://visualstudio.microsoft.com/vs/

**Reply by Rohan-Zanjal** [reply]
Yaa, I get it,
Thank you.
